examples_MATLAB/maxwells2d.m

```diff
@@ -0,0 +1,162 @@
+clc;
+clear all;
+close all;
+
+addpath('mole-master/mole-master/mole_MATLAB/');
```

Comment:
@hareesh-cs - Please change this to: addpath('../mole_MATLAB');
to be consistent with the rest of the examples and to work correctly on a system where MOLE has yet to be used.